packages/vue/src/Overlay/Modal/OcModalPage.vue

```diff
@@ -51,13 +55,12 @@ const onClickOutside = async () => {
     v-if="modelValue"
     class="fixed w-screen z-[1007] bg-white top-0 left-0 h-full min-h-screen bg-black/[.45] flex items-center justify-center"
   >
-    <div class="w-full h-full absolute top-0 left-0 z-[1007]" @click="onClickOutside" />
```

Comment:
why did you remove the `onClickOutside`?

Reply:
No need to handle. The modal page is persisted